proofs/tla/test/unit/relay/relayTests.tla

```diff
@@ -0,0 +1,164 @@
+------------------------------ MODULE relayTests -------------------------------
+
+EXTENDS FiniteSets, TLC, utils
+ASSUME LET T == INSTANCE TLC IN T!PrintT("relayTests")
+
+CONSTANTS WalMaxRowsTest
+ASSUME WalMaxRowsTest \in Nat
+
+--------------------------------------------------------------------------------
+\* Imports
+--------------------------------------------------------------------------------
+
+CONSTANTS Servers
+ASSUME Cardinality(Servers) > 1
+VARIABLES relay, box, msgs, wal
+allVars == <<relay, box, msgs, wal>>
+INSTANCE relay
+
+--------------------------------------------------------------------------------
+\* Unit tests
+--------------------------------------------------------------------------------
+
+LOCAL toName == CHOOSE x \in Servers: TRUE
+LOCAL fromName == CHOOSE x \in Servers: x # toName
+
+LOCAL RelayDefaultState == [
+    msgsToSend |-> << >>,
+    msgsToReceive |-> << >>,
+    relayQueue |-> << >>,
+    lastAck |-> EmptyAck(Servers),
+    raftMsg |-> EmptyGeneralMsg,
+    sentLsn |-> 0,
+    txQueue |-> << >>,
+    \* RO variables.
+    toInstance |-> toName,
+    walRows |-> << >>
+]
+
+----------------------------
+\* Test RelayRead operator |
+----------------------------
```

Comment:
Could you please make section comments consistent? In modules above section comment looks as below:

![image](https://github.com/user-attachments/assets/d02bfdb7-658f-415c-aadf-d3750e19d285)


here and below

Reply:
Dropped these subsections)